README.md

```diff
@@ -1,12 +1,123 @@
 # SmartKickers
 
-Definition of Done
-
-- Working code
-- Created code need to be merged
-- Tests implemented
-- Tests passed
-- Code review by devs
-- Sanity check with mentors
-- Acceptance criteria fulfilled 
-- Work presented and accepted
+## Overview
+
+SmartKickers is a project to make your kickers more immersive. It uses Ximea camera to track the ball. 
+
+The following repository contains: 
+* Golang server for processing the data.
+* React client to display game information.
+
+You will also need SmartKickersAI provided by the developers per request. It contains:
+* Ball tracking software.
+* Node server responsible for sending data to the Golang server.
+
+Link to SmartKickerAI
+https://github.com/HackYourCareer/SmartKickersAI.
+
+## Features
+
+* Keeping track of the score.
+
+## Prerequirements
+
+* [python 3.10.5](https://www.python.org/downloads/macos/)
+* [node 16.16.0](https://nodejs.org/en/download/)
+* npm 8.11.0
+* macOS 12.4
+## Requirements
+
+* [ximea camera drivers](https://www.ximea.com/support/wiki/apis/ximea_macos_software_package)
+* opencv-python 
+
+    ```bash 
+    pip3 install open-cv
+    ```
+
+* numpy
+
+    ```bash 
+    pip3 install numpy
+    ```
+
+* imutils
+
+    ```bash 
+    pip3 install imutils
+    ```
+
+## Installation
+
+We are using macOS to build and run everything locally for now.
+
+> **NOTE:** Ensure you have also SmartKickersAI source code.
+
+1. Clone the repositories.
+
+    ```bash 
+    git clone git@github.com:HackYourCareer/SmartKickers.git
+
+    git clone git@github.com:HackYourCareer/SmartKickersAI.git
+    ```
+
+2. Run the requirementsCheck.sh script to check if your system fulfils all requirements.
```

Comment:
Couple of things:
1. Both repositories should have their own `requirementsCheck.sh` scripts that include information about their own dependencies.
2. In the `1. Clone the repositories.` you say to clone both repositories but then the users don't know where should they look for the `requirementsCheck.sh script`. You should guide them and add information where should they look for e.g:
```bash
cd SmartKickers
./requirementsCheck.sh
```

Reply:
1. We will add this later. For now, this script is just for the end user to check that he has all the dependencies installed.
2. Fixed.